src/pages/[...slug].tsx

```diff
@@ -62,41 +67,35 @@ const componentsMapping = {
   tutorial: tutorialsComponents,
 } as const
 
-interface Params extends ParsedUrlQuery {
```

Comment:
Can't we keep using this type? we could reuse it in `GetStaticPaths<Params>` and `GetStaticProps<Props, Params>`

Reply:
@pettinarip ah we can keep this original type if we want to throw type errors at the type signature and not when it is passed into the generic. 

Now thinking about it, I tend to do this. So I shouldn't have removed this! 😅

Reply:
Added it back in. It's in a different area of the file.